runtime/mainnet/src/config/monetary.rs

```diff
@@ -0,0 +1,439 @@
+		const CENTS: Balance = DOLLARS / 100; // 100_000_000
+		const MILLI_CENTS: Balance = CENTS / 1_000; // 100_000
+
+		// https://github.com/polkadot-fellows/runtimes/blob/e220854a081f30183999848ce6c11ca62647bcfa/relay/polkadot/constants/src/lib.rs#L36
+		fn relay_deposit(items: u32, bytes: u32) -> Balance {
+			items as Balance * 20 * DOLLARS + (bytes as Balance) * 100 * MILLI_CENTS
+		}
+
+		// https://github.com/polkadot-fellows/runtimes/blob/e220854a081f30183999848ce6c11ca62647bcfa/system-parachains/constants/src/polkadot.rs#L70
+		fn system_para_deposit(items: u32, bytes: u32) -> Balance {
+			relay_deposit(items, bytes) / 100
+		}
+
+		assert_eq!(deposit(2, 64), system_para_deposit(2, 64))
+	}
+
+	#[test]
+	fn units_are_correct() {
+		// UNIT should have 10 decimals
+		assert_eq!(UNIT, 10_000_000_000);
+		assert_eq!(MILLI_UNIT, 10_000_000);
+		assert_eq!(MICRO_UNIT, 10_000);
+
+		// fee specific units
+		assert_eq!(fee::CENTS, 100_000_000);
+		assert_eq!(fee::MILLI_CENTS, 100_000);
+	}
+
+	#[test]
+	fn transaction_byte_fee_is_correct() {
+		assert_eq!(fee::TRANSACTION_BYTE_FEE, 50_000);
+	}
+
+	#[test]
+	fn balances_stores_account_balances_in_system() {
+		assert_eq!(
+			TypeId::of::<<Runtime as pallet_balances::Config>::AccountStore>(),
+			TypeId::of::<System>(),
+		);
+	}
+
+	#[test]
+	fn balances_uses_u128_balance() {
+		assert_eq!(
+			TypeId::of::<<Runtime as pallet_balances::Config>::Balance>(),
+			TypeId::of::<u128>(),
+		);
+	}
+
+	#[test]
+	fn balances_done_slash_handler_does_not_have_callbacks() {
+		assert_eq!(
+			TypeId::of::<<Runtime as pallet_balances::Config>::DoneSlashHandler>(),
+			TypeId::of::<()>(),
+		);
+	}
+
+	#[test]
+	fn balances_dust_removal_handler_resolves_to_treasury_account() {
+		assert_eq!(
+			TypeId::of::<<Runtime as pallet_balances::Config>::DustRemoval>(),
+			TypeId::of::<ResolveTo<TreasuryAccount, Balances>>(),
+		);
+
+		new_test_ext().execute_with(|| {
+			let existential_deposit =
+				<<Runtime as pallet_balances::Config>::ExistentialDeposit>::get();
+			let who = AccountId::from([1u8; 32]);
+			let beneficiary = AccountId::from([2u8; 32]);
+			let call = RuntimeCall::Balances(pallet_balances::Call::transfer_allow_death {
+				dest: sp_runtime::MultiAddress::Id(beneficiary),
+				value: UNIT + (existential_deposit / 2),
+			});
+			Balances::set_balance(&who, existential_deposit + UNIT);
+			Balances::set_balance(&TreasuryAccount::get(), UNIT);
+
+			// `who`'s balance goes under ED.
+			assert_ok!(call.dispatch(RuntimeOrigin::signed(who.clone())));
+			// `who` has been dusted.
+			assert_eq!(Balances::free_balance(&who), 0);
+			System::assert_has_event(RuntimeEvent::Balances(pallet_balances::Event::DustLost {
+				account: who,
+				amount: existential_deposit / 2,
+			}));
+			// Treasury balance equals its initial balance + the dusted amount from `who`.
+			assert_eq!(
+				Balances::free_balance(&TreasuryAccount::get()),
+				UNIT + existential_deposit / 2
```

Comment:
```suggestion
				existential_deposit + existential_deposit / 2
```

if you agree with using ED value instead of UNIT for the initial treasury balance.

Reply:
As above.